enterprise/plugins/com.liferay.ide.kaleo.core/META-INF/MANIFEST.MF

```diff
@@ -6,14 +6,14 @@ Bundle-Version: 3.1.2.qualifier
 Bundle-Activator: com.liferay.ide.kaleo.core.KaleoCore
 Bundle-Vendor: Liferay, Inc.
 Require-Bundle: org.eclipse.core.runtime,
- org.eclipse.sapphire.modeling;bundle-version="[9,10)";visibility:=reexport,
- org.eclipse.sapphire.modeling.xml;bundle-version="[9,10)";visibility:=reexport,
+ org.eclipse.sapphire.modeling;visibility:=reexport,
```

Comment:
Actually we could leave the sapphire ranges, since sapphire normally breaks things.